docs/en/connector-v2/sink/TDengine.md

```diff
@@ -0,0 +1,69 @@
+# TDengine
+
+> TDengine sink connector
+
+## Description
+
+Used to write data to TDengine. You need to create stable before running seatunnel task
+
+## Key features
+
+- [x] [parallelism](../../concept/connector-v2-features.md)
+- [ ] [support user-defined split](../../concept/connector-v2-features.md)
```

Comment:
Refer to docs/en/concept/connector-v2-features.md, sink only has two features exactly-once and cdc